olp-cpp-sdk-authentication/src/TokenRequest.h

```diff
@@ -41,6 +47,17 @@ class TokenRequest {
   TokenRequest(
       const std::chrono::seconds& expires_in = std::chrono::seconds(0));
 
+  /**
+   * @brief Sets the expiration time.
+   *
+   * @param expires_in (Optional) The number of seconds left before the new
+   * access token expires.
+   *
+   * @return A reference to *this.
```

Comment:
No need to add `@return` to setters. Please remove here and in other setters below.

Reply:
done